content/hi/blue_green_deployment.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Blue Green Deployment
```

Comment:
Would it be good to translate `Blue Green Deployment` into Hindi too? 😊

There are other cases like:
- `DevOps(डेवॉप्स)`
- `निरंतर एकीकरण (Continuous Integration)`
- `निरंतर डिलीवरी (CD)`

Reply:
I thought of keeping it in English because I was not sure but IG the same `Blue Green Deployment` should be written in Hindi like `ब्लू ग्रीन डिप्लॉयमेंट` rather than translating it. (need suggestion)

Reply:
After reviewing other localizations I have decided to write it in Hindi.